pkg/controller/constrainttemplatestatus/constrainttemplatestatus_controller_test.go

```diff
@@ -321,6 +375,19 @@ func verifyTStatusCount(ctx context.Context, c client.Client, expected int) func
 	}
 }
 
+func verifyTStatusCreated(ctx context.Context, c client.Client, expected bool) func() error {
+	return func() error {
+		ct := &v1beta1.ConstraintTemplate{}
+		if err := c.Get(ctx, types.NamespacedName{Name: "denyall"}, ct); err != nil {
```

Comment:
maybe not hardcode this but instead pass in a var?

Reply:
`verifyTByPodStatusCount` has it hardcoded, that's why I've kept it hardcoded :)

I'd suggest to add a const with a name instead of passing a var or hardcoding, would that be fine?